docs/Inputs.md

```diff
@@ -152,6 +163,19 @@ import { AutocompleteInput, ReferenceInput } from 'react-admin'
 
 **Tip**: React-admin's `<AutocompleteInput>` has only a capital A, while material-ui's `<AutoComplete>` has a capital A and a capital C. Don't mix up the components!
 
+### Properties
+Prop | Required/Optional | Type | Default | Description
+---|---|---|---|---
+`choices` | Required | `Object[]` | - | List of items to autosuggest
+`resource` | Required | `string` | - | The resource working on. This field is passed down by wrapped components like `Create` and `Edit`.  
+`source` | Required |  `string` | - | Name of field to edit, it's type should correspond to the type retrieved from `optionValue` 
+`allowEmpty` | Optional | `boolean` | `false` | If `false` and the searchText typed did not match any suggestion, the searchText will revert to the current value when the field is blurred. If `true` and the `searchText` is set to `''` then the field will set the input value to `null`.
```

Comment:
Can you either remove this line or specify that it will be passed from the `ReferenceInput`. Users should not specify `allowEmpty` directly on the `AutocompleteInput`